File: dart/scripts/hello_branch.dart
void main() {
    var helloWorld = 'Hello, world!';

    if (helloWorld == 'Good morning') {
        print('morning');
    } else if (helloWorld.contains('Hello')) {
        print('noon');
    } else {
        print('night');
    }

    switch (helloWorld) {
        case 'Hello, world!':
            print('nonn');
            break;
        case 'Good morning, world!':
            print('morning');
            break;
        default:
            print('night');
    }
}

File: elixir/scripts/hello_branch.exs
hello_world = "Hello, world!"

_ =
  cond do
    hello_world == "Good morning" ->
      "morning"

    String.contains?(hello_world, "Hello") ->
      "noon"

    true ->
      "night"
  end
  |> IO.puts()

_ =
  case hello_world do
    "Hello, world!" ->
      "noon"

    "Good morning, world!" ->
      "morning"

    _ ->
      "neight"
  end
  |> IO.puts()
